.env.template

```
@@ -67,11 +67,15 @@ PROMETHEUS_AUTHORIZATION_TOKEN=
 # Note that anonymous players don't have any TTL limit because their data is stored in local storage, not in Redis database.
 PLAYER_VARIABLES_MAX_TTL=-1
 
-# FEATURE FLAGS
+# MAP EDITOR SETTINGS
+ENABLE_MAP_EDITOR=true
+# If you want to allow only some users to access the map editor, you can set the list of authorized users here, email separated by commas. (Only possible if OIDC is enabled)
+# Leave blank if you want to allow all users to access the map editor.
+USERS_ALLOWED_TO_MAP_EDITOR=
```

Could you rename this variable to:

```suggestion
MAP_EDITOR_ALLOWED_USERS=
```